code/datums/status_effects/debuffs.dm

```diff
@@ -463,6 +463,7 @@
 		qdel(src)
 	else
 		add_bleed(-1)
+		owner.apply_damage(10)
```

Comment:
Make sure this doesnt massively increase the damage output of the bloodletter knife and gangrel cleave ability (both use bleed effect and probably shouldnt be able to deal passive 10 damage)

Reply:
Oh god, will do.

Reply:
Tested and addressed by adding a faction check on the bleeding mobs, as long as there are no humans belonging to either boss or mining faction they wouldn't die horribly in the span of a few seconds.